liquid_node/configuration.py

```diff
@@ -201,6 +202,32 @@ def __init__(self):
 
         self.timestamp = int(time.time())
 
+        self.app_description = defaultdict(lambda: "No description", {
```

Comment:
this doesn't need to be an instance variable, it can be local to the constructor